sdks/go/test/integration/driver/driver.go

```diff
@@ -69,6 +69,13 @@ func main() {
 		// {"flatten:dup", primitives.FlattenDup()},
 		{"reshuffle:reshuffle", primitives.Reshuffle()},
 		{"reshuffle:reshufflekv", primitives.ReshuffleKV()},
+		{"window:sums", func() *beam.Pipeline {
```

Comment:
Is this driver still getting used somewhere? I think we disabled the gradle commands that were running this on Jenkins, and if so it might be time to delete this code.

Reply:
Seems reasonable, but definitely not in this PR. I wasn't entirely sure TBH.